apparmor.d/groups/virt/calico

```diff
@@ -0,0 +1,26 @@
+abi <abi/3.0>,
```

Comment:
You need to add the file header (same for all new profiles):
```
# apparmor.d - Full set of apparmor profiles
# Copyright (C) 2022 You
# SPDX-License-Identifier: GPL-2.0-only
```

Reply:
Ah, I missed that one. Added.